2020-02-Spring/3_trees/README.md

```diff
@@ -70,21 +70,91 @@ Go to [Solution](#s2)   [Top](#top)
 <!-- Don't remove -->
 <a name="p3"/>
 
-### 3. PROBLEM 3 TODO :bug:
+### 3. Recover Binary Search Tree
 
-Source: TODO :bug:
+[LeetCode](https://leetcode.com/problems/recover-binary-search-tree/)
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Two elements of a binary search tree (BST) are swapped by mistake.
```

Comment:
Maybe I'm just being dumb, but I do not understand how you know which two elements are swapped. Is the point to figure that out?

Reply:
same issue above.

Reply:
Yeah, I think a big part of this problem is you have to know the binary search tree invariants well enough to think about using inorder traversal to check if the tree is valid.

Reply:
Good idea. That would work.